app/go_to_date/main.py

```diff
@@ -0,0 +1,18 @@
+from fastapi import FastAPI, Request
```

Comment:
Please choose an existing directory

Reply:
Didn't quite understand what did you mean?
what should I change?

Reply:
The path of the file should be app/(EXISTING_DIRECTORY_HERE)/main.py, not `go_to_date`

Reply:
I still don't understand, should I put my feature's files directly in the "app" directory?
don't really understand what you mean by existing directory?

Reply:
There are many directories that are already exists: `routers`, `internal` etc.

Choose one of them which fits the best, according to the schema described [here](https://fastapi.tiangolo.com/tutorial/bigger-applications/), and move this file to there.

Don't create your own directory for each feature :)

Reply:
Ok, so if I understand it right I need to put my python file at the internal folder (and maybe change its name to go_to_date), and the content of static folder I should put in the static folder app/static, and the same goes for the templates folder?